.github/workflows/create_promptflow_release_tag.yml

```diff
@@ -0,0 +1,35 @@
+name: Create promptflow release tag
+
+on:
+  workflow_dispatch:
```

Comment:
Shall we restrict this to only run on release branch(since it will modify _version.py)?

Reply:
good catch, add a check on prefix